src/com/amazon/corretto/crypto/provider/AesGcmSpi.java

```diff
@@ -294,6 +294,10 @@ protected synchronized void engineInit(int opMode, Key key, SecureRandom secureR
     protected synchronized void engineInit(
         int jceOpMode, Key key, AlgorithmParameterSpec algorithmParameterSpec, SecureRandom secureRandom
     ) throws InvalidKeyException, InvalidAlgorithmParameterException {
+	if (key == null) {
+	    throw new InvalidKeyException("Key can't be null");
+	}
```

Comment:
Check indent? This looks like it might not be the same level as other code in this method.

Reply:
Yep, my bad. It's fixed now.